docs/quickstart.html

```diff
@@ -233,19 +233,16 @@ <h4 class="anchor-heading">
             Edit the <code class="language-bash">config/connect-standalone.properties</code> file, add or change the <code>plugin.path</code> configuration property match the following, and save the file:
         </p>
 
-        <pre class="brush: bash;">
-&gt; echo "plugin.path=libs/connect-file-{{fullDotVersion}}.jar"</pre>
+        <pre class="line-numbers"><code class="language-bash">$ echo "plugin.path=libs/connect-file-{{fullDotVersion}}.jar"</code></pre>
```

Comment:
Should the command be `echo "plugin.path=libs/connect-file-3.8.0.jar" >> config/connect-standalone.properties`, otherwise the echo command just print the line

Reply:
I believe you are correct. I was checking the history of the file, and it seems it has been always wrong.

Reply:
pushed the fix